src/content/self-hosted/administration/configuration.mdx

```diff
@@ -217,6 +217,18 @@ The `mapping` fields are optional. Use them to configure the mapping between Okt
 
 > Your provider needs to support the [UserInfo endpoint](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo) in order to be used with Okteto. This authentication option follows the OpenID standard, and it has been validated with Okta, PingIdentity, and GitLab.
 
+#### Token
+
+Token-based is the default authentication method.
+This method permits users to validate their identity by employing a randomly generated token comprising eight characters.
+The token-based authentication approach is an optimal choice for testing/evaluation scenarios, where access to Okteto is restricted to a single user.
+
+```yaml
+auth:
+  token:
+    enabled: true
```

Comment:
we need to add instructions on how to retrieve the token (or pass a pre-existing value, if we support that)

Reply:
This is what we show after `helm install`:
```
1. Create the following DNS record, pointing it to the NGINX controller service External-IP:
    "*.pablo.dev.okteto.net"

You can retrieve the External IP by running:
    kubectl get service -l=app.kubernetes.io/name=ingress-nginx,app.kubernetes.io/component=controller --namespace=okteto

2. Create the following DNS record, pointing it to the Buildkit service External-IP:
    "buildkit.pablo.dev.okteto.net"

You can retrieve the External IP by running:
    kubectl get service -l=app.kubernetes.io/instance==pablo,app.kubernetes.io/component=buildkit --namespace=okteto

3. Once you create both DNS entries you can access your Okteto instance at this URL:
    https://okteto.pablo.dev.okteto.net/#/login?token=d9f2fd6c

In order to use the Okteto CLI with the instance, you need to execute the following commands:
- Install the Okteto CLI:
    curl https://get.okteto.com -sSfL | sh

- Configure the context of your Okteto CLI
    okteto context use https://okteto.pablo.dev.okteto.net --token d9f2fd6c
``` 

Would you show that output? or would you put it here with different wording?

Reply:
I added a sentence on this, let me know what think